app/uk/gov/hmrc/eoricommoncomponent/frontend/CdsErrorHandler.scala

```diff
@@ -67,7 +67,6 @@ class CdsErrorHandler @Inject() (
     exception match {
       case sessionTimeOut: SessionTimeOutException =>
         // $COVERAGE-OFF$Loggers
```

Comment:
Can you please remove coverage statements as well?

Reply:
The coverage statements are now removed.